### PR TITLE
Add ```GetAllVertexCoordinates3DNC``` function

### PR DESCRIPTION
**Pull request by schnellecom**
Add a function to return all coordinates saved in a printrecord as a list.

### CODE REVIEW

File: gap/PolygonalComplexes/animating.gd [hunk 1]
```diff
@@ -143,12 +143,15 @@ DeclareOperation( "SetVertexCoordinates3DNC", [IsSimplicialSurface, IsList, IsRe
 #! The 3D-coordinates of vertex <A>vertex</A> has to have the format <A>[x,y,z]</A>.
 #! If the format of the <A>coordinates</A> is not correct, then an error is shown.
 #! This can happen, if the NC version is used to store the 3D-coordinates.
-#! The NC-version does not check the coordinate format saved in the print record.
+#! The NC-version does not check the coordinate format saved in the print record. 
+#! The <A>GetAllVertexCoordinates3DNC</A> command returns all coordinates from the print record without checking them as a list.
```
Comment (MeikeWeiss) [inline gap/PolygonalComplexes/animating.gd:147]: I think "method" would be a more appropriate word than "command".

File: gap/PolygonalComplexes/animating.gi [hunk 2]
```diff
@@ -88,6 +88,14 @@ InstallMethod( GetVertexCoordinates3DNC,
 RedispatchOnCondition(GetVertexCoordinates3DNC, true, [IsTwistedPolygonalComplex, IsPosInt, IsRecord],
     [IsSimplicialSurface], 0);
 
+InstallMethod( GetAllVertexCoordinates3DNC,
+    "for a simplicial surfaceand a record",
```
Comment (MeikeWeiss) [inline gap/PolygonalComplexes/animating.gi:92]: A whitespace is missing here.

**Review by MeikeWeiss — CHANGES_REQUESTED** [review]
Looks good!

File: gap/PolygonalComplexes/animating.gd (hunk 1 above)
Comment (schnellecom) [reply gap/PolygonalComplexes/animating.gd]: See [772bfc6](https://github.com/gap-packages/SimplicialSurfaces/pull/259/commits/772bfc640c06f07b2470aadb228b7fdb9ca35aa5)

File: gap/PolygonalComplexes/animating.gi (hunk 2 above)
Comment (schnellecom) [reply gap/PolygonalComplexes/animating.gi]: See [772bfc6](https://github.com/gap-packages/SimplicialSurfaces/pull/259/commits/772bfc640c06f07b2470aadb228b7fdb9ca35aa5)

**Review by MeikeWeiss — APPROVED** [review]

**Pull request merged**